tuneup.py

```diff
@@ -23,25 +35,33 @@ def read_movies(src):
 def is_duplicate(title, movies):
     """Case insensitive search within a list"""
     for movie in movies:
-        if movie.lower() == title.lower():
+        if movie == title:
```

Comment:
Case insensitivity was removed during refactoring. Why? This could impact functionality.